Repositories/DataContext.cs

```diff
@@ -0,0 +1,11 @@
+﻿using DomainModels;
+using System;
+using System.Collections.Generic;
+
+namespace Repositories
+{
+    public class DataContext
+    {
+        public List<Item> Items { get; set; }
```

Comment:
Please instantiate the Items property because by default it is null and you cannot modify the contents of the list

Reply:
Thank you for reviewing  - this was updated.